bitnami/zookeeper/templates/statefulset.yaml

```diff
@@ -91,7 +91,7 @@ spec:
           {{- $electionPort := int .Values.service.electionPort }}
           {{- $releaseNamespace := .Release.Namespace }}
           {{- $zookeeperFullname := include "zookeeper.fullname" . }}
-          {{- $zookeeperHeadlessServiceName := printf "%s-%s" $zookeeperFullname "headless" | trunc 24  }}
+          {{- $zookeeperHeadlessServiceName := printf "%s-%s" $zookeeperFullname "headless" | trunc 30  }}
```

Comment:
actually, you can increase the limit to 63.

Reply:
Alright 👍 
I have updated the trunc value from 24 to 63.

But is there any specific reason for 63? or is it just a big magic number?